src/backends/torch/torchsolver.cc

```diff
@@ -291,6 +310,14 @@ namespace dd
                 std::string bin;
                 std::string test_name;
                 bestfile.open(bestfilename, std::ios::in);
+                if (!bestfile.is_open())
```

Comment:
Does this mean resuming fails if there's a file but no identified best model ?

Reply:
yes, and it is corrected now: resume can be done even if not test has been done yet